godotproject/Level0AttractMode.gd

```diff
@@ -0,0 +1,50 @@
+extends Node2D
+
+# Each level has to have a name in it now, this will be displayed on the high score screen
+const level_name := "Morning Meadow"
+
+const inhibit_hud = true
+const inhibit_pause = true
+
+export (bool) var record = false
+
+const SPIKE := 3
+const STAR := 12
+
+export (PackedScene) var Spike
+export (PackedScene) var Star
```

Comment:
the spike stuff can be deleted as you never interact with them

Reply:
Same comment as above, it works now so no need to change imo. This file was just copied from the Level0 and modified enough to work.